lib/chef_zero/endpoints/principal_endpoint.rb

```diff
@@ -30,13 +30,22 @@ def get(request)
           end
         end
         if json
-          json_response(200, {
+          principal_data = {
             'name' => name,
             'type' => type,
             'public_key' => FFI_Yajl::Parser.parse(json)['public_key'] || PUBLIC_KEY,
             'authz_id' => '0'*32,
             'org_member' => org_member
-          })
+          }
+
+          response_data =
+            if request.api_v0?
+              principal_data
+            else
+              { "principals" => [ principal_data ] }
```

Comment:
If there is both a user and a client with the given name, should we return them both?